app/frontend/src/components/forms/manage/Subscription.vue

```diff
@@ -131,9 +131,11 @@ export default {
         }
       } catch (error) {
         this.addNotification({
-          message:
-            'An error occurred while attempting to update the settings for this form.',
-          consoleError: `Error updating settings for ${this.form.id}: ${error}`,
+          message: this.$t('trans.subscribeEvent.saveSettingsErrMsg'),
+          consoleError: this.$t('trans.subscribeEvent.updateSettingsConsoleErrMsg', {
```

Comment:
The unit tests are failing because of the Prettier checks - please run the unit tests and it will automatically run Prettier on the files.